.github/copilot-instructions.md

```
@@ -4,7 +4,15 @@
 - **Use consistent naming conventions, file structure, and architecture patterns** as described in `PLANNING.md`.
 - **Use uv** (the package manager) whenever executing Python commands, including for unit tests. eg `uv run example.py` or `uv run test tests/test_example.py`.
 - **Add package dependencies** to `pyproject.toml` using the `uv add` command as needed.
-
+### 🌳 Git & PR Management (Graphite)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)